doc/nrf/config_and_build/bootloaders/bootloader_downgrade_protection.rst

```diff
@@ -99,6 +105,6 @@ Downgrade protection using MCUboot
 
 To enable anti-rollback protection with monotonic counter for MCUboot, set the following configurations using sysbuild:
```

Comment:
For line 93:

See the :ref:`zephyr:sysbuild_kconfig_namespacing` section for information on how to set options for built images in Sysbuild.